ee/server/service/hosts.go

```diff
@@ -239,6 +250,18 @@ func (svc *Service) enqueueUnlockHostRequest(ctx context.Context, hostID uint, l
 	// TODO(mna): must save the script's execution uuid into host_mdm_actions...
 	// Ideally that would be in the same DB transaction.
 
+	if err := svc.ds.NewActivity(
+		ctx,
+		vc.User,
+		fleet.ActivityTypeUnlockedHost{
+			HostID:          host.ID,
+			HostDisplayName: host.DisplayName(),
+			HostPlatform:    host.Platform,
```

Comment:
We can use the same activity type for view pin/unlock and the host platform signals whether the UI should display the viewed unlock pin message (macOS) or the basic unlock message (not macOS)